Content.Shared/Physics/CollisionGroup.cs

```diff
@@ -28,6 +28,7 @@ public enum CollisionGroup
 
         MobMask = Impassable | MobImpassable | VaultImpassable | SmallImpassable,
         ThrownItem = MobImpassable | Impassable,
+        SwingAttack = Opaque | Impassable | MobImpassable,
```

Comment:
Probably don't need opaque, I would also just leave it on the melee-weapon directly if it's only used in the one spot.